Marlin/ultralcd.cpp

```diff
@@ -1077,6 +1077,137 @@ void kill_screen(const char* lcd_msg) {
+      0x60,0x00,0x00, // 011000000000000000000000
+      0x20,0x00,0x00, // 001000000000000000000000
+      0x10,0x00,0x00, // 000100000000000000000000
+      0x0c,0x0c,0x00, // 000011000000110000000000
+      0x07,0xf8,0x00  // 000001111111100000000000
+    };
+
+    const unsigned char ccw_ubl_bmp[] PROGMEM = { //AVR-GCC, WinAVR
+      0x01,0xfe,0x00, // 000000011111111000000000
+      0x03,0x03,0x00, // 000000110000001100000000
+      0x04,0x00,0x80, // 000001000000000010000000
+      0x08,0x00,0x40, // 000010000000000001000000
+      0x18,0x00,0x60, // 000110000000000001100000
+      0x10,0x00,0x20, // 000100000000000000100000
+      0x7c,0x00,0x20, // 011111000000000000100000
+      0x38,0x00,0x20, // 001110000000000000100000
+      0x10,0x00,0x20, // 000100000000000000100000
+      0x00,0x00,0x20, // 000000000000000000100000
+      0x00,0x00,0x20, // 000000000000000000100000
+      0x00,0x00,0x60, // 000000000000000001100000
+      0x00,0x00,0x40, // 000000000000000001000000
+      0x00,0x00,0x80, // 000000000000000010000000
+      0x03,0x03,0x00, // 000000110000001100000000
+      0x01,0xfe,0x00  // 000000011111111000000000
+    };
+
+
+    const unsigned char up_arrow_ubl_bmp[] PROGMEM = { //AVR-GCC, WinAVR
+      0x06,0x00, // 000001100000
+      0x0F,0x00, // 000011110000
+      0x1F,0x80, // 000111111000
+      0x3F,0xC0, // 001111111100
+      0x06,0x00, // 000001100000
+      0x06,0x00, // 000001100000
+      0x06,0x00, // 000001100000
+      0x06,0x00, // 000001100000
+      0x06,0x00, // 000001100000
+      0x06,0x00, // 000001100000
+      0x06,0x00, // 000001100000
+      0x06,0x00, // 000001100000
+      0x06,0x00  // 000001100000
+    };
+
+    const unsigned char down_arrow_ubl_bmp[] PROGMEM = { //AVR-GCC, WinAVR
+      0x06,0x00, // 000001100000
+      0x06,0x00, // 000001100000
+      0x06,0x00, // 000001100000
+      0x06,0x00, // 000001100000
+      0x06,0x00, // 000001100000
+      0x06,0x00, // 000001100000
+      0x06,0x00, // 000001100000
+      0x06,0x00, // 000001100000
+      0x06,0x00, // 000001100000
+      0x3F,0xC0, // 001111111100
+      0x1F,0x80, // 000111111000
+      0x0F,0x00, // 000011110000
+      0x06,0x00  // 000001100000
+    };
+
+    const unsigned char offset_bedline_ubl_bmp[] PROGMEM = { //AVR-GCC, WinAVR
+      0xFF,0xFF,0xFF // 111111111111111111111111
+    };
+
+    const unsigned char nozzle_ubl_bmp[] PROGMEM = { //AVR-GCC, WinAVR
+      0x7F,0x80, // 0111111110000000
+      0xFF,0xC0, // 1111111111000000
+      0xFF,0xC0, // 1111111111000000
+      0xFF,0xC0, // 1111111111000000
+      0x7F,0x80, // 0111111110000000
+      0x7F,0x80, // 0111111110000000
+      0xFF,0xC0, // 1111111111000000
+      0xFF,0xC0, // 1111111111000000
+      0xFF,0xC0, // 1111111111000000
+      0x3F,0x00, // 0011111100000000
+      0x1E,0x00, // 0001111000000000
+      0x0C,0x00  // 0000110000000000
+    };
+
+        void _lcd_ubl_zoffset_overlay(float zprobe_zoffset) {
```

Comment:
I think this function could be shared between BABYSTEP_ZPROBE_GFX_OVERLAY and AUTO_BED_LEVELING_UBL. Although they would end up sharing a static variable for keeping track of turn direction, it should still work ok as the graphics would be updated as soon as the user started to turn the dial.